mix.exs

```diff
@@ -12,8 +12,10 @@ defmodule Spell.Mixfile do
      deps: deps,
      aliases: aliases,
      docs: docs,
-     preferred_cli_env: ["test.unit": :test,
+     preferred_cli_env: ["test.all": :test,
+                         "test.unit": :test,
                          "test.integration": :test,
+                         "test.integration.all": :test,
```

Comment:
Can we simplify this that "test.integration" always run for all defined serializer? Than we don't need "test.integration.all" and "test.all".